implementations/elixir/ockam/ockam_services/lib/services/provider.ex

```diff
@@ -60,6 +60,42 @@ defmodule Ockam.Services.Provider do
   end
 
   def get_service_child_specs({service_name, service_args}, providers) do
+    case get_named_service_child_specs(service_name, service_args, providers) do
+      {:ok, specs} ->
+        {:ok, specs}
+
+      {:error, {:unknown_service, service_name}} ->
+        case is_service_module?(service_name) do
+          true ->
+            {:ok, [module_child_spec(service_name, service_args)]}
+
+          false ->
+            {:error, {:unknown_service, service_name}}
+        end
+    end
+  end
+
+  defp is_service_module?(service_name) do
+    case Code.ensure_loaded(service_name) do
+      {:module, _module} ->
+        function_exported?(service_name, :child_spec, 1)
```

Comment:
What would the consequences be if we provided a module that defines a `child_spec/1` but is not a "service"?

Reply:
Then it's a service. There is no clear definition of what it means to be a service, there was an assumption before that it would have some special properties, but as it turns out we would be better off with just supervised processes.
This PR is a first step towards this direction. Maybe we could go all the way and delete the services all together, but I would rather do that in stages.